src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -0,0 +1,340 @@
+    getCreateStudyDialogFormDefaultValues,
+} from './create-study-dialog-utils';
+import {
+    API_CALL,
+    CASE_FILE,
+    CASE_NAME,
+    CASE_UUID,
+    DESCRIPTION,
+    FORMATTED_CASE_PARAMETERS,
+    STUDY_NAME,
+} from '../../utils/field-constants';
+import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
+import CustomMuiDialog from '../commons/custom-mui-dialog/custom-mui-dialog';
+import { ErrorInput, FieldErrorAlert, TextInput } from '@gridsuite/commons-ui';
+import StudyNamePrefilledInput from './study-name-prefilled-input';
+
+const MAX_FILE_SIZE_IN_MO = 100;
+const MAX_FILE_SIZE_IN_BYTES = MAX_FILE_SIZE_IN_MO * 1024 * 1024;
+const STRING_LIST = 'STRING_LIST';
+
+const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
+    const intl = useIntl();
+    const { snackError } = useSnackMessage();
+    const dispatch = useDispatch();
+
+    const [caseFileLoading, setCaseFileLoading] = useState(false);
+
+    const activeDirectory = useSelector((state) => state.activeDirectory);
+    const selectedDirectory = useSelector((state) => state.selectedDirectory);
+    const userId = useSelector((state) => state.user.profile.sub);
+
+    const createStudyFormMethods = useForm({
+        mode: 'onChange',
+        defaultValues: getCreateStudyDialogFormDefaultValues({
+            directory: activeDirectory,
+            studyName: providedExistingCase?.elementName,
+        }),
+        resolver: yupResolver(createStudyDialogFormValidationSchema),
+    });
+
+    const {
+        setValue,
+        formState: { errors },
+        setError,
+        clearErrors,
+        getValues,
+        watch,
+    } = createStudyFormMethods;
+
+    const studyName = watch(STUDY_NAME);
+
+    const isValid = isObjectEmpty(errors);
+
+    // callbacks
+    const handleApiCallError = useCallback(
+        (error) => {
+            if (error.status === HTTP_UNPROCESSABLE_ENTITY_STATUS) {
+                setError(`root.${API_CALL}`, {
+                    type: 'invalidFormatOrName',
+                    message: intl.formatMessage({ id: 'invalidFormatOrName' }),
+                });
+            } else if (error.message.includes(HTTP_CONNECTION_FAILED_MESSAGE)) {
+                setError(`root.${API_CALL}`, {
+                    type: 'serverConnectionFailed',
+                    message: intl.formatMessage({
+                        id: 'serverConnectionFailed',
+                    }),
+                });
+            } else {
+                setError(`root.${API_CALL}`, {
+                    type: 'apiCall',
+                    message: intl.formatMessage({ id: 'error.message' }),
+                });
+            }
+        },
+        [intl, setError]
+    );
+
+    const getCurrentCaseImportParams = useCallback(
+        (uuid) => {
+            getCaseImportParameters(uuid)
+                .then((result) => {
+                    result.parameters = result.parameters?.map((parameter) => {
```

Comment:
You deleted a comment line, put it back please
```suggestion
// sort possible values alphabetically to display select options sorted
                    result.parameters = result.parameters?.map((parameter) => {
```